libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -821,6 +851,9 @@ def write_mcu_config(f):
 #define CH_CFG_USE_MAILBOXES FALSE
 #define CH_CFG_USE_FACTORY FALSE
 #define CH_CFG_USE_MEMCORE FALSE
+#define CH_CFG_USE_DELEGATES FALSE
```

Comment:
Do we want these even in firmware by default?

Reply:
you're right. I've changed the defaults on this and a few others